### PR TITLE
Added volumes, volumeMounts and additional labels to the chart

### PR DESCRIPTION
**Pull request by RezaTehraniAPT**
#### What does this PR do?
Adding volumes, volume mounts, pod labels and additional labels (which will be applied to all the resources defined in the chart) 

#### Developer Done List

- [ ] Tests Added/Updated
- [ ] Updated README.md
- [ ] Verified backward compatible
- [ ] Verified database migrations will not be catastrophic
- [ ] Considered Security, Availability and Confidentiality

#### Reviewer/Approval Done List

- [ ] Tests Pass Locally
- [ ] CI Build Passes
- [ ] Verified README.md is updated
- [ ] Verified changes are backward compatible
- [ ] Reviewed impact to Security, Availability and Confidentiality (if issue found, add comments and request changes)

### CODE REVIEW

File: charts/metrics-agent/values.yaml
Comment (daniel-spray) [inline charts/metrics-agent/values.yaml:1]: Please also increment the version here https://github.com/cloudability/metrics-agent/blob/9e9ac70becfbe7ce5cc3c6912f4488039501892a/charts/metrics-agent/values.yaml#L27

File: charts/metrics-agent/templates/deployment.yaml [hunk 1]
```diff
@@ -52,6 +53,14 @@ spec:
             {{- toYaml .Values.resources | nindent 12 }}
           livenessProbe:
             {{- toYaml .Values.livenessProbe | nindent 12 }}
+          {{- with .Values.volumeMounts }}
```
Comment (daniel-spray) [inline charts/metrics-agent/templates/deployment.yaml:56]: You also tested applying the chart with no volumes and no volume mounts and there were no errors?

Comment (RezaTehraniAPT) [reply charts/metrics-agent/templates/deployment.yaml:56]: Yes, tested this use-case as well.

File: charts/metrics-agent/values.yaml
Comment (RezaTehraniAPT) [reply charts/metrics-agent/values.yaml:1]: Fixed

**Review by daniel-spray — APPROVED** [review]

**Pull request merged**